StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -659,13 +660,15 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
     func selectNetworkedAccounts(
         selectedAccountIds: [String],
         clientSecret: String,
-        consumerSessionClientSecret: String
+        consumerSessionClientSecret: String,
+        consentAcquired: Bool?
     ) -> Future<FinancialConnectionsInstitutionList> {
-        let parameters: [String: Any] = [
+        var parameters: [String: Any] = [
             "selected_accounts": selectedAccountIds,
             "client_secret": clientSecret,
             "consumer_session_client_secret": consumerSessionClientSecret,
         ]
+        parameters["consent_acquired"] = consentAcquired
```

Comment:
[example Stripe.js](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/pull/23488/files#diff-60871d17beee6942de7c86d06c3f88155b664634dd06b96c30c0e2479e6e7fb4R1001)

Reply:
This is essentially step #2 of this PR: use `acquireConsentOnPrimaryCtaClick` and pass it as `consentAcquired`

Reply:
Should we do something like this, rather than possibly passing `nil` to the parameter?
```suggestion
        if let consentAcquired {
                parameters["consent_acquired"] = consentAcquired
        }
```

Reply:
Do you have a example/source for it? I've always believed a `nil` is a way of clearing keys

One case where it does not is assigning an object: ex. NSNull object

I tried to pull in some docs (https://developer.apple.com/documentation/swift/dictionary):

<img width="943" alt="Screenshot 2024-08-01 at 10 33 15 AM" src="https://github.com/user-attachments/assets/b92c5e0a-8cef-47fd-b4c0-af5a21f91ebb">